### PR TITLE
Adding code coverage artefacts and netlify hosted coverage report

### PR DESCRIPTION
**Pull request by CptHash**
After a push on the 'dev' branch the report will be available here: https://front-coverage.netlify.app/

☑️ TDD Security
✅ Minimal Testing
✅ Accurate assignees / reviewers.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -18,20 +16,31 @@ jobs:
...
-      #needs: [build]
+      needs: [build]
       steps:
-        - name: Quickmetrics
-          uses: screendriver/quickmetrics-action@v1
-          with:
-            name: fake-deploy-front
-            api-key: ${{ secrets.QUICKMETRICS_API_KEY }}
         - name: Invoke deployment hook
+          if: github.ref == 'refs/heads/main'
```
Comment (louischristner) [inline .github/workflows/node.js.yml:43]: What happen when this condition is false ? Does it fail the job ?

Comment (CptHash) [reply .github/workflows/node.js.yml:43]: It will only skip the deployment hooks. Like that it will be trigger only during a push on main branch

**Review by louischristner — APPROVED** [review]
Really happy for code coverage, I approve !

**Review by hgvo9 — APPROVED** [review]
That's really good, I approve

**Pull request merged**